src/internal/m365/collection/drive/collections_test.go

```diff
@@ -2839,6 +2910,62 @@ func (suite *OneDriveCollectionsUnitSuite) TestGet() {
 				rootFolderPath2: true,
 			},
 		},
+		{
+			name:   "duplicate previous paths in metadata",
```

Comment:
I'm unsure if we should add another test (or expand this one) to have two drives, one of which has duplicate paths. I think it would be good coverage to have, though of course does come with more maintenance requirements

Reply:
Sure, that's easy to do.